english/wikipedia/wiki.md

```diff
@@ -8,4 +8,6 @@ You are blind to all but your exploration of the known and unknown universe.
 
 [Behold!](http://en.wikipedia.org)
 
+You are so impressed with Wikipedia, you decide that you might even like to [donate](https://donate.wikimedia.org/w/index.php?title=Special:FundraiserLandingPage&country=XX&uselang=en&utm_medium=sidebar&utm_source=donate&utm_campaign=C13_en.wikipedia.org) some money to help keep Wikipedia running.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.